Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -313,9 +313,20 @@ public extension UIImage {
     ///
     /// - Parameters:
     ///   - base64String: a base-64 `String`, representing the image
-    convenience init?(base64String: String) {
+    ///   - scale: The scale factor to assume when interpreting the image data created from the base-64 string. Applying a scale factor of 1.0 results in an image whose size matches the pixel-based dimensions of the image. Applying a different scale factor changes the size of the image as reported by the `size` property.
+    convenience init?(base64String: String, scale: CGFloat = 1.0) {
         guard let data = Data(base64Encoded: base64String) else { return nil }
-        self.init(data: data)
+        self.init(data: data, scale: scale)
+    }
+    
+    /// SwifterSwift: Create a new image from a URL
+    ///
+    /// - Parameters:
+    ///   - url: a `URL`, representing the image
```

Comment:
The `URL` doesn't represent the image, it locates it.

Reply:
haha the issue with copy / paste @guykogus